clipping_index_usage.py

```diff
@@ -0,0 +1,24 @@
+import os
+from river_segmentation.crop_image.index_processer import IndexProcesser
+
+def main():
+    path_project = r"path/to/your/river/photos"
+    # Create instance of the index processer
+    processer = IndexProcesser()
+    # The index file must be in a subfolder within the TIF file's folder 
+    # The structure must be as follows: PHOTOS_TIF/Index/index_file_name.shp
+    # IMPORTANT: The Index folder must start with capital I
+    processer.index = os.path.join(path_project,"Index/index_file_name.shp")
+    processer.photos = path_project
+    processer.reproject_index("2947") #Select the EPSG code based on the TIF image projection
```

Comment:
Put the crs projection in global variable. It may changes

Reply:
I will change, that's the idea. Good idea to have it as global